### ISSUE TITLE
OnUpdate Cancel Event is not working

### ISSUE THREAD

**Issue by lampewebdev** [replit/kaboom#459]
### Version

v2000.1

### What browsers are you seeing the problem on?

Chrome

### What happened?

When pressing a for 120 frames, I get the following error:
```bash
Uncaught TypeError: this.exists is not a function
```
from destroy() in kaboom.js

### What's the expected behavior?

Canceling an onUpdate should work

### Minimum reproducable code
https://replit.com/@lampewebdev/test#code/main.js

```ts
import kaboom from "kaboom";

// initialize context
kaboom();

// load assets
loadSprite("bean", "sprites/bean.png");

const Dialog = () => {
  let state = "idle";
  let onUp = null;
  let counter = 0;

  const startDraingwDialog = () => {
    return onUpdate(() => {
      console.log(state);
      counter += dt();
      if(counter > 2) {
        state = "end"
      }
    });
  };

  onKeyDown("a", () => {
    if(state === "idle") {
      state = "drawing";
      onUp = startDraingwDialog();
      
    }
    if(state === "end") {
      onUp();
    }
  })
};

Dialog();
```

### Live demo

https://test.lampewebdev.repl.co/

### Is there an existing issue for this?

- [X] I've searched and there is no existing issue for this bug

**Comment by lampewebdev** [comment]
A smaller repo would be
```ts
import kaboom from "kaboom";
kaboom();
const up = onUpdate(() => {})
up();
```

**Comment by slmjkdbtl** [comment]
Thanks. Fixed in v2000.1.7
